faker/providers/person/es_CA/__init__.py

```diff
@@ -0,0 +1,65 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from ..es_ES import Provider as PersonProvider
+
+
+class Provider(PersonProvider):
+    """
+    Adds popular Catalan names to the spanish ones.
```

Comment:
The added names come from a proper source, but by adding the Spanish one we're effectively build our own dataset, which goes outside of the scope of Faker. Could we stick to _just_ what's on the data source?

Reply:
Spain is complicated from an external point of view. It has Spanish, Catalan, Galician, Basque, Occitan languages under one country. Each region has a lot of mixing and crossover, i don't think that having _JUST_ catalan names is correct, since you will find a lot of "normal" spanish names in Catalunya.

Reply:
Wouldn't such complexity be reflected in the data collected by the statistics?

Reply:
Thats a VERY good point. I've amended the list using two sources: baby names, popular names in the region.